src/app/drivingLicence/fields.js

```diff
@@ -13,7 +13,7 @@ const dvlaValidatorObj = {
     "middleNames",
     "surname",
     "dateOfBirth",
-    "drivingLicenceNumber",
```

Comment:
I think we should keep driving licence number in as we will use it for some future name validations